Seems that the '5' enforces the major version and it refuses to find
system's assimp-6.0.2 (which is perfectly usable given Qt 6.9.2 bundles
that version without it).

We don't package older ones, so drop the check as a quick workaround.

https://bugreports.qt.io/browse/QTBUG-137996
--- a/src/plugins/assetimporters/configure.cmake
+++ b/src/plugins/assetimporters/configure.cmake
@@ -16 +16 @@
-qt_find_package(WrapQuick3DAssimp 5.1.6 PROVIDED_TARGETS WrapQuick3DAssimp::WrapQuick3DAssimp MODULE_NAME assetimporters QMAKE_LIB quick3d_assimp)
+qt_find_package(WrapQuick3DAssimp PROVIDED_TARGETS WrapQuick3DAssimp::WrapQuick3DAssimp MODULE_NAME assetimporters QMAKE_LIB quick3d_assimp)
@@ -38 +38 @@
-                   PACKAGES PACKAGE WrapQuick3DAssimp 5.1.6)
+                   PACKAGES PACKAGE WrapQuick3DAssimp)
